.pre-commit-config.yaml

```diff
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
```

Comment:
If we change a revision and the style changes, would that mean we get bloated commits as it would modify / beautify touch all the files in a given branch? Meaning it would touch more files that have been originally modified?

Reply:
not automatically

locally, precommit only lints files within the pending commit

you could run a precommit on all files (manually), and set this up as a separate pr